### PR TITLE
Added a README for Linkled Lists

### PR DESCRIPTION
**Pull request by Zanark**
## ISSUE NUMBER

Issue #100 
## SHORT DESCRIPTION

**Added a README.md for Linked List.** 
Please review it and I would be glad if you could send me a feedback on how the README.md can be changed if necessary and if my format of the file is acceptable because I will create other README.md(s) in a similar format.

## TESTING
The README.md will be displayed below the list of files on the GitHub Repository (in the Linked List folder) page.

### CODE REVIEW

File: Competitive Coding/Linked List/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Linked List operations
+
+**This folder contains files that have the following algorithms**
+
+ 1. All BASIC Linked List operations on a Linked List based on *Pointers*
+   
+    * Adding elements at the *end* of a Linked List.
+    * Adding elements at the *begining* of a Linked List.
```
Comment (prateekiiest) [inline Competitive Coding/Linked List/README.md:8]: Like give a link to the subpart of the Linked List programming demonstrating this.

**Review by prateekiiest — CHANGES_REQUESTED** [review]
can you add the links to each of the Linked List programs

File: Competitive Coding/Linked List/README.md (hunk 1 above)
Comment (Zanark) [reply Competitive Coding/Linked List/README.md:8]: Okay I will update the README and send it to you soon

**Comment by Zanark** [comment]
I sincerely apologize for closing the pull request 
I had some trouble with the forked repo so had to create a new branch instead
I will update the README and will send you a new PR soon

**Pull request closed without merging**